### DISCUSSION TITLE
The terminal block does not fit long responses

### DISCUSSION THREAD

**Discussion by dybo-io** [postmanlabs/postman-flows#11] [Defect]
We have an API request which has a response of about 1200-1300 lines. After running and viewing in the terminal block, it only shows the last 200-300 lines (or in other words, cannot scroll up past that). Sorry, but cannot share the response or a screenshot as it's confidential to the company.

**Comment by saswatds** [comment]
@dybo-io I have gone ahead a created an issue to increase the character limit we have on terminal #12
